godel_path_planning/src/trajectory_planning.cpp

```diff
@@ -0,0 +1,188 @@
+#include "godel_path_planning/trajectory_planning.h"
+
+#include <moveit/robot_trajectory/robot_trajectory.h>
+#include <moveit/robot_model_loader/robot_model_loader.h>
+
+#include "descartes_moveit/moveit_state_adapter.h"
+#include "descartes_trajectory/cart_trajectory_pt.h"
+#include "descartes_planner/dense_planner.h"
+
+#include <tf_conversions/tf_eigen.h>
+
+
+// Anonymous namespace
+namespace
+{
+  const static double TOOL_POINT_DELAY = 0.75;
+
+  // Create a descartes RobotModel for graph planning
+  descartes_core::RobotModelPtr createRobotModel(const moveit::core::RobotStatePtr robot_state,
+                                                 const std::string& group_name,
+                                                 const std::string& tool_frame,
+                                                 const std::string& world_frame,
+                                                 const uint8_t iterations)
```

Comment:
The iterations argument doesn't need the const attribute if it's being passed by value.